pkg/prowgen/prowgen_test.go

```diff
@@ -54,11 +55,20 @@ func TestGeneratePodSpec(t *testing.T) {
 			},
 			targets: []string{"target"},
 		},
+		{
+			description: "private job with skip_cloning false",
```

Comment:
test case for the opposite?

Reply:
The `private job` test case is the opposite.